Assets/HoloToolkit/Input/Prefabs/MixedRealityCameraParent.prefab

```diff
@@ -614,8 +614,6 @@ MonoBehaviour:
   GoBackButtonName: Fire2
   LeftJoystickX: ControllerLeftStickX
   LeftJoystickY: ControllerLeftStickY
-  RightJoystickX: ControllerRightStickX
```

Comment:
Yep, this was left over from when we removed the ability to look around with the right stick and added bumper rotation.